src/components/StandardCardTable/StandardCardTable.jsx

```diff
@@ -38,7 +44,23 @@ const StandardCardTable = ({ cards, setCards, ...props }) => {
             </Button>
           )}
 
-          <Button onClick={onOpen} rounded={4} boxShadow="base">
+          <Button
+            onClick={() => {
+              if (!user || !user.isAdmin) {
+                unauthorizedToast({
+                  title: "Unauthorized!",
+                  description: "You must log in as an admin.",
+                  status: "error",
+                  duration: 3000,
+                  isClosable: true,
+                });
+                return;
+              }
+              onOpen();
```

Comment:
Honestly, with how many times this exact snippet is being used, it might even be worth making a custom wrapper function that abstracts away the process of checking if user is logged in or not. The code could look something like
```jsx
<Button onclick={ifAdmin(onOpen)}>
    Admin Button
</Button>
```
where `ifAdmin` returns a function. If that seems doable, it might be worth doing.

Either way, let's extract to function again